invenio.cfg

```diff
@@ -312,6 +316,24 @@ CDS_REDIRECTION_COLLECTIONS_MAPPING = {
     "CERN Students Projects": "",  # uuid
 }
 
+RDM_SEARCH = {
+    **deepcopy(RDM_SEARCH),
+    "query_parser_cls": QueryParser.factory(
+        mapping={
+            "experiment": "custom_fields.cern\:experiments.title.en",
```

Comment:
Open question: when writing in the search box `experiment`, do you think that the user searches for the `id`, or for the `title`?

Reply:
I would say title, more human friendly field